js/app.js

```diff
@@ -339,8 +340,10 @@ function onMessageArrived(message) {
         devices.deviceList[device_id]["stats_uptime"] = payload;
       } else if (topic[2] === "interval") {
         devices.deviceList[device_id]["stats_interval"] = payload;
-      } else if (topic[2] === "freeheap") {
+      } else if (topic[2] === "free-heap") {
```

Comment:
According to the [specification](https://github.com/homieiot/convention/blob/develop/extensions/documents/homie_legacy_stats_extension.md) it should be `freeheap` not `free-heap`.

Reply:
You have right, I didn't see the specification. But that is related to Homie 4.0 or also to 3.0.1?

As you can see here: https://github.com/elbowz/homie-esp8266/blob/c4ed8ec33e3c5f3ce5f651a2b903be28e865dacf/src/Homie/Boot/BootNormal.cpp#L198 now Homie-esp8266 works with `free-heap`.

But I guess the right way is fix on Homie-esp8266

Reply:
In 4.0 stats are an extension rather than part of the core spec. The stats extension specification is what I linked above.